packages/block-library/src/heading/edit.js

```diff
@@ -36,6 +38,16 @@ function HeadingEdit( {
 		} ),
 		style: mergedStyle,
 	} );
+	useEffect( () => {
+		const newAnchor = maybeUpdateAnchor(
```

Comment:
What happens if I entered the anchor manually (I want to force an anchor), will this force me to update it anytime the content changes?

Reply:
Auto-generated anchors are prefixed with `wp-`, and `maybeUpdateAnchor` checks if the anchor starts with that. So manually entered anchors won't be changed, and they are just skipped. Only auto-generated anchors change automatically with the content.

Reply:
Makes sense